src/clzones.cpp

```diff
@@ -42,6 +42,7 @@
 #include "vehicle.h"
 #include "visitable.h"
 #include "vpart_position.h"
+#include "options.h"
```

Comment:
Don't need this header any more I don't think.

Reply:
Removed